packages/apps/storybook/.storybook/main.js

```diff
@@ -9,6 +13,14 @@ module.exports = {
     "@itwin/storybook-auth-addon",
     "../src/addon/register.js",
   ],
+  reactOptions: { fastRefresh: true },
+  framework: '@storybook/react-webpack5',
```

Comment:
I get an error when running storybook: 
<img width="201" alt="image" src="https://github.com/iTwin/admin-components-react/assets/1904889/60232e79-ae5a-4307-82c7-a4c493e0255a">
Is that `@storybook/react-webpack5` a package ? Because it is not listed in the package.json.

Reply:
Pull the latest changes, should be fixed